app/assets/stylesheets/variables.scss

```diff
@@ -105,6 +105,16 @@ $font-size-h6: $font-size-larger;
   }
 }
 
+@mixin font-size-very-small {
```

Comment:
perhaps we should stick to bootstrap convention and name it font-size-xs or something similar?